docs/VnVPlan/VnVPlan.tex

```diff
@@ -69,141 +69,157 @@ \section{Symbols, Abbreviations, and Acronyms}
+    \item Clarity: the document is written in a clear, concise, and unambiguous manner
+\end{itemize}
 
 \subsection{Design Verification Plan}
-
-\wss{Plans for design verification}
-
-\wss{The review will include reviews by your classmates}
-
-\wss{Create a checklists?}
+The SRS will be verified with an informal walk-through by stakeholders (individuals who are members of low-income households in Hamilton and students from McMaster University) and peers of the SFWRENG 4G06 course who will provide ad-hoc review as well as an inspection by the development team. The review will address the following issues and be guided by the checklist:
+\begin{itemize}
+    \item Design adequacy and conformance to software requirements: assess whether the design adequately meets the specified requirements (SRS)
+    \item Internal consistency: different components of the design are coherent
+    \item External consistency: the design aligns with external interfaces, standards, and environments
+    \item Feasibility: whether the design can be practically realized within technical and resource constraints
+    \item Maintenance: whether the design can be easily updated, modified, or repaired
+\end{itemize}
 
 \subsection{Verification and Validation Plan Verification Plan}
-
-\wss{The verification and validation plan is an artifact that should also be
-verified.  Techniques for this include review and mutation testing.}
-
-\wss{The review will include reviews by your classmates}
-
-\wss{Create a checklists?}
+The VnV plan will be verified with an informal walk-through by stakeholders (individuals who are members of low-income households in Hamilton and students from McMaster University) and peers of the SFWRENG 4G06 course who will provide ad-hoc review as well as an inspection by the development team. The reviews will address the following issues and be guided by the checklist:
+\begin{itemize}
+    \item Completeness: the VnV plan covers all necessary documentation, functional requirements, and non-functional requirements
+    \item Feasibility: whether the actions that the VnV plan details can be practically realized within technical and resource constraints
+    \item Clarity: the document is written in a clear, concise, and unambiguous manner
+\end{itemize}
 
 \subsection{Implementation Verification Plan}
-
-\wss{You should at least point to the tests listed in this document and the unit
-  testing plan.}
-
-\wss{In this section you would also give any details of any plans for static
-  verification of the implementation.  Potential techniques include code
-  walkthroughs, code inspection, static analyzers, etc.}
+The verification of the implementation will be done with static and dynamic tests. Dynamic tests will be done according to sections 4 and 5 of this document. For static tests, the development team will conduct a code walk-through to check for incorrect code along with using the linter and coding standards detailed in section 3.6.
```

Comment:
Maybe explicitly mention the use of surveys and focus groups with stakeholders as well for both dynamic and static tests, especially for NFRs?